### PR TITLE
OAuth2 permissions

### PR DESCRIPTION
**Pull request by topolik**
Permissions

### CODE REVIEW

**Comment by csierra** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by csierra** [comment]
merged... thx!

File: modules/apps/oauth2-provider/oauth2-provider-scopes-impl/src/main/java/com/liferay/oauth2/provider/scopes/impl/scopedescriptor/DefaultScopeDescriptor.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	public String describe(String scope, Locale locale) {
 
 		ResourceBundle resourceBundle =
-			_resourceBundleLoader.loadResourceBundle(locale);
+			_resourceBundleLoader.loadResourceBundle(locale.toString());
```
Comment (csierra) [inline modules/apps/oauth2-provider/oauth2-provider-scopes-impl/src/main/java/com/liferay/oauth2/provider/scopes/impl/scopedescriptor/DefaultScopeDescriptor.java]: LanguageUtil.toLanguageId() or LocaleUtil.toLanguageId()

**Comment by topolik** [comment]
Do you want me to send the fix?

-- tom +

**Pull request closed without merging**